docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  server:
+    build: .
+    environment:
+      IMS_DB_HOST_NAME: mysql
+      IMS_DB_DATABASE: ims
+      IMS_DB_USER_NAME: ims
+      IMS_DB_PASSWORD: ims
+    ports:
+      - "8080:80"
+    depends_on:
+      mysql:
+        condition: service_healthy
+    links:
+      - mysql
+  mysql:
+    image: mariadb:10.5.17
+    environment:
+      MARIADB_DATABASE: ims
+      MARIADB_USER: ims
+      MARIADB_PASSWORD: ims
+      MARIADB_ROOT_HOST: "%"
+      MARIADB_ROOT_PASSWORD: ims-root
+    volumes:
+      - ./.mysql/data/:/var/lib/mysql
```

Comment:
Not a blocker. 
My personal preference is to keep docker specific directories & configs like this in a `docker` or `.docker` sub-directory to keep the project root from getting too busy even if it's just our local environments.

Reply:
Done